assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -92,13 +97,33 @@ const toggleSortLookup = {
   [SORT_DIRECTION.ASC]: SORT_DIRECTION.DESC,
 };
 
+const LastModifiedTableHeaderCell = styled(TableHeaderCell)`
+  min-width: 160px;
+`;
+
+const AuthorTableHeaderCell = styled(TableHeaderCell)`
+  min-width: 110px;
+`;
+
 export default function StoryListView({
   filteredStories,
   storySort,
   handleSortChange,
   handleSortDirectionChange,
   sortDirection,
+  tags,
+  categories,
+  users,
```

Comment:
Curious your thoughts on just grabbing these from context here instead of passing them as props from the my-stories view since they are just objects getting referenced by Ids found in the `filteredStories` (which we should probably rename now that its not doing any kind of filtering, but that's beside the point).